app/models/job_invocation_composer.rb

```diff
@@ -191,6 +263,21 @@ def compose
     self
   end
 
+  def trigger(raise_on_error = false)
+    generate_description
```

Comment:
Generating the description here fails to correctly retrieve the input values and generates description without them.